features/map.go

```diff
@@ -159,6 +140,84 @@ func isStorageMap(mt ebpf.MapType) bool {
 	case ebpf.SkStorage, ebpf.InodeStorage, ebpf.TaskStorage:
 		return true
 	}
-
 	return false
 }
+
+// HaveMapFlag probes the running kernel for the availability of the specified map flag.
+//
+// See the package documentation for the meaning of the error return value.
+func HaveMapFlag(flag uint32) (err error) {
```

Comment:
There is no way to discover which flags the library supports probing for.

I think we should have a `type MapFlag uint32` here with aliases to `unix.BPF_F_*` consts defined for the flags that we support probing for.

`MapFlag` could also be in `ebpf` proper, but that raises the question of circular imports again.

Reply:
I agree with you but I just matched what the rest of the API is using, like https://pkg.go.dev/github.com/cilium/ebpf#Map.Flags. Maybe we should change this as well (but this would warrants another PR)

Reply:
I think we can add `sys.MapFlags` and re-export that via type aliases.